frontend/src/main.js

```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue'
+import firebase from "firebase";
+import App from './App.vue'
+import './registerServiceWorker'
+import router from './router'
+import store from './store'
+
+Vue.config.productionTip = false
+
+let app = "";
+const config = {
```

Comment:
Can you add the link to firebase console from where you obtained below values.

Reply:
Where exactly do you want me to add the firebase console link?